TriblerGUI/widgets/trustpage/js/main.js

```diff
@@ -32,7 +32,7 @@ var state = {
     request_pending: false,
     x: width / 2,
     y: height / 2,
-    focus_pk: "self",
+    focus_pk: "10",
```

Comment:
I would guess this focus node is because of debugging purposes. Please change this.